synapse/lib/cell.py

```diff
@@ -208,67 +220,67 @@ def setCellUser(self, iden):
         except Exception:
             pass
 
-    @adminapi
+    @adminapi(log=True)
     async def delAuthUser(self, name):
         await self.cell.auth.delUser(name)
         await self.cell.fire('user:mod', act='deluser', name=name)
 
-    @adminapi
+    @adminapi(log=True)
     async def addAuthRole(self, name):
         role = await self.cell.auth.addRole(name)
         await self.cell.fire('user:mod', act='addrole', name=name)
         return role.pack()
 
-    @adminapi
+    @adminapi(log=True)
     async def delAuthRole(self, name):
         await self.cell.auth.delRole(name)
         await self.cell.fire('user:mod', act='delrole', name=name)
 
-    @adminapi
+    @adminapi(log=True)
     async def getAuthUsers(self, archived=False):
         '''
         Args:
             archived (bool):  If true, list all users, else list non-archived users
         '''
         return [u.name for u in self.cell.auth.users() if archived or not u.info.get('archived')]
 
-    @adminapi
+    @adminapi(log=True)
     async def getAuthRoles(self):
         return [r.name for r in self.cell.auth.roles()]
 
-    @adminapi
+    @adminapi(log=True)
     async def addUserRule(self, name, rule, indx=None, gateiden=None):
         user = await self.cell.auth.reqUserByName(name)
         retn = await user.addRule(rule, indx=indx, gateiden=gateiden)
         return retn
 
-    @adminapi
+    @adminapi(log=True)
     async def addRoleRule(self, name, rule, indx=None, gateiden=None):
         role = await self.cell.auth.reqRoleByName(name)
         retn = await role.addRule(rule, indx=indx, gateiden=gateiden)
         return retn
 
-    @adminapi
+    @adminapi(log=True)
     async def delUserRule(self, name, rule, gateiden=None):
         user = await self.cell.auth.reqUserByName(name)
         return await user.delRule(rule, gateiden=gateiden)
 
-    @adminapi
+    @adminapi(log=True)
     async def delRoleRule(self, name, rule, gateiden=None):
         role = await self.cell.auth.reqRoleByName(name)
         return await role.delRule(rule, gateiden=gateiden)
 
-    @adminapi
+    @adminapi(log=True)
     async def setUserAdmin(self, name, admin, gateiden=None):
         user = await self.cell.auth.reqUserByName(name)
         await user.setAdmin(admin, gateiden=gateiden)
 
-    @adminapi
+    @adminapi(log=True)
     async def setRoleAdmin(self, name, admin, gateiden=None):
         role = await self.cell.auth.reqRoleByName(name)
         await role.setAdmin(admin, gateiden=gateiden)
 
-    @adminapi
+    @adminapi(log=True)
```

Comment:
I think log only the mutating auth stuff.

Reply:
ah yeah this was me going through all the auth stuff but the non-mutating stuff can easily be silenced.